Emby.Server.Implementations/ApplicationHost.cs

```diff
@@ -267,6 +268,8 @@ protected ApplicationHost(
                 ? Environment.MachineName
                 : ConfigurationManager.Configuration.ServerName;
 
+        public string RestoreBackup { get; set; }
```

Comment:
Think `RestoreBackupPath` is a better name for this. I know it is in the docs but when I saw this property dangling here I wondered if it was going to be a magic string